lib/graphql/query/context.rb

```diff
@@ -195,6 +211,10 @@ def received_null_child
         @value = nil
       end
 
+      def scoped_merge!(hash)
+        @scoped_context = @scoped_context.merge(hash)
```

Comment:
Should this mutate with `merge!`?

Reply:
See https://github.com/rmosolgo/graphql-ruby/pull/2634#discussion_r356284461 🙃